src/api/Controllers/BoreholeController.cs

```diff
@@ -149,56 +151,135 @@ private IQueryable<Borehole> GetBoreholesWithIncludes()
     /// Exports the details of up to <see cref="MaxPageSize"></see> boreholes as a CSV file. Filters the boreholes based on the provided list of IDs.
     /// </summary>
     /// <param name="ids">The list of IDs for the boreholes to be exported.</param>
-    /// <returns>A CSV file containing the details specified boreholes.</returns>
+    /// <returns>A CSV file containing the details of the specified boreholes.</returns>
     [HttpGet("export-csv")]
     [Authorize(Policy = PolicyNames.Viewer)]
     public async Task<IActionResult> DownloadCsvAsync([FromQuery][MaxLength(MaxPageSize)] IEnumerable<int> ids)
```

Comment:
@MiraGeowerkstatt @Lehats Zu besprechen: Die Uploads erfolgen in einem separaten `UploadController` und die Downloads hier im `BoreholeController`. Wir vermischen hier nun einiges habe ich das Gefühl und sind nicht konsequent. Entweder brauchts einen `DownloadController` oder wir packen alles hier in den `BoreholeController`. Was meint ihr?

Reply:
Das stimmt. Da der `UploadController` aber auch schon einige Logik beinhaltet, denke ich würde der `BoreholeController` zu gross werden. Ich schlage daher einen separaten Controller vor. Frage an euch: Ist es überhaupt sinnvoll von Upload und Download zu sprechen oder macht Import/Export mehr Sinn?